mmengine/model/wrappers/fully_sharded_distributed.py

```diff
@@ -207,8 +244,8 @@ def val_step(self, data: dict) -> List[BaseDataElement]:
         Returns:
             List[BaseDataElement] or dict: The predictions of given data.
         """
-        inputs, data_sample = self.module.data_preprocessor(data, False)
-        return self(inputs, data_sample, mode='predict')
+        data = self.module.data_preprocessor(data, False)
+        return self._run_forward(data, mode='predict')  # type: ignore
```

Comment:
Suggest directly to call `self.module.test_step`

Reply:
This is not feasible in FSDP. Typically we should call FSDP's `forward` function to rebuild parameters. Otherwise error will occur.